content/guides/component-testing/quickstart-angular.md

```diff
@@ -0,0 +1,125 @@
+---
+title: 'Quickstart: Angular'
+---
+
+<CtBetaAlert></CtBetaAlert>
+
+To follow along with this guide, you'll need an Angular CLI application.
+
+The quickest way to get started writing component tests for Angular is to use
+the [Angular CLI](https://angular.io/cli).
+
+To create an Angular project:
+
+1. Install the Angular CLI
+
+```bash
+npm install -g @angular/cli
+```
+
+2. Create a new Angular application:
+
+```bash
+ng new my-awesome-app
+```
+
+3. Add Cypress
+
+```bash
+npm install cypress -D
+```
+
+4. Open it and follow the Launchpad's prompts!
+
+```bash
+npx cypress open
+```
+
+## Configuring Component Testing
+
+When you run Cypress for the first time in the project, the Cypress app will
+prompt you to set up either **E2E Testing** or **Component Testing**. Choose
+**Component Testing** and step through the configuration wizard.
+
+<DocsImage 
+  src="/img/guides/component-testing/select-test-type.png" 
+  caption="Choose Component Testing"> </DocsImage>
+
+The Project Setup screen automatically detects your framework, which is Angular.
+Cypress Component Testing uses your existing development server config to render
+components, helping ensure your components act and display in testing the same
+as they do in production.
+
+<DocsImage 
+  src="/img/guides/component-testing/project-setup-angular.png" 
+  caption=""> </DocsImage>
+
+Next, the Cypress setup will detect your framework and generate all the
+necessary configuration files, and ensure all required dependencies are
+installed.
+
+<DocsImage 
+  src="/img/guides/component-testing/scaffolded-files-angular.png" 
+  caption="The Cypress launchpad will scaffold all of these files for you">
```

Comment:
Looks like these lines are borrowed from the other framework templates. So, this will be more correct one 😄 

```suggestion
  caption="The Cypress launchpad will scaffold all of these files for you.">
```